cesql/spec.md

```diff
@@ -353,6 +357,25 @@ left operand of the OR operation evalues to `true`, the right operand MUST NOT b
 
 #### 3.7. Type casting
 
+A CESQL engine MUST support the following type casts:
```

Comment:
Yeah I see your point. Personally, I think "returning" makes more sense because it's not like an exception that is raised can be caught somewhere - it is just a second return value you get on every evaluation (which may be null/nil). WDYT? If you think "returning" makes more sense, I'll open a PR to change that everywhere, otherwise I'll switch these to be "raising" an error.

Reply:
Sorry, I wasn't clear - I'm fine with the "raising" terminology, but the problem is that these casts don't say what the non-error part of the return is when there's a problem. If I cast "BOGUS" to Boolean, an error will be raised... but what's the value of the expression?

Reply:
Oh I see your point - let me add some clarification here.

Reply:
I added some clarification, could you recheck when you have time @jskeet ?

Reply:
Yup, that definitely works.